tests/mppsolar_tests/test_outputs.py

```diff
@@ -0,0 +1,23 @@
+import importlib
+import unittest
+from pathlib import Path
+
+HERE = Path( __file__).parent
+OUTPUTS_DIR = HERE.parent.parent / "mppsolar" / "outputs"
+OUTPUTS = sorted(
+    file.stem for file in OUTPUTS_DIR.glob("*.py")
+    if file.stem != "__init__"
+)
```

Comment:
We can't use `list_output()` here, for two reasons:
- it doesn't work from tests, it seems the fact that it's not packaged is messing the whole logic
- more importantly, we need to really try all outputs, without skipping ones that are not available

Same remark for the protocol test file below.